lib/amazing_print/ext/active_record.rb

```diff
@@ -55,7 +55,7 @@ def awesome_active_record_instance(object)
                  end
                end
              end
-      "#{object} " + awesome_hash(data)
+      [object.to_s, awesome_hash(data)].join(' ')
```

Comment:
Is there a particular advantage to creating an array and then joining? It feels like this might be a bit more complicated that just adding strings together.

Reply:
My thought process behind it was `awesome_active_record_class` joins strings via an array because there are 3 attributes:
https://github.com/amazing-print/amazing_print/blob/244242cce50e051e93547d1d15e1fdea58af0ae0/lib/amazing_print/ext/active_record.rb#L78

I felt it looked nicer in the event other formatters added additional attributes to the output:
```
name + base + etc + etc2 + ... +  awesome_hash(data)
```

But you're right about overcomplicating it now. I'll change it to `String#+` 😁👍

Reply:
Looking over this file again and neighboring files, I'm starting to agree with you :smile_cat: 
`Array#join` definitely reads nicer beyond two elements and I like the idea of consistency.